src/ProductView/Diagram/LineChart.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import Chart from "chart.js";
+import {argosConfig} from '../../config/argosConfig.js';
+
+class LineChart extends Component {
+    
+    static sortEventsByTime(elementA, elementB, timeStampAttribute) {
+        let datePartsA = elementA[timeStampAttribute].split("T")[0];
+        let datePartsB = elementB[timeStampAttribute].split("T")[0];
+        datePartsA = datePartsA.split("-");
```

Comment:
Assigning an array to a string variable makes it harder to read. I suggest using a new const instead.